pkg/command/context.go

```diff
@@ -101,14 +99,23 @@ var createCtxCmd = &cobra.Command{
     tanzu context create mgmt-cluster --kubeconfig path/to/kubeconfig --kubecontext kubecontext
 
     # Create a TKG management cluster context by using the provided CA Bundle for TLS verification:
```

Comment:
I feel there should be some mention of how the CLI would attempt to detect the context type where possible. 
Coupled with that we either show what you have below and mentioned that `--type` might be omittable, OR show examples without the `--type` then mention that `--type` might need to be added in some cases.

Reply:
Thanks! Updated the examples with and without `--type` and mentioned that `--type` is optional and if not provided would be inferred from the endpoint in some cases.